packages/lesswrong/server/rateLimitUtils.ts

```diff
@@ -205,22 +210,51 @@ export async function rateLimitDateWhenUserNextAbleToPost(user: DbUser): Promise
   // fetch the posts from within the maxTimeframe
   const postsInTimeframe = await getPostsInTimeframe(user, maxHours);
 
-  const rateLimitInfos = getPostRateLimitInfos(user, postsInTimeframe, modRateLimitHours, manualPostRateLimit, recentKarmaInfo);
+  const features = await getFeatures(user)
+
+  const rateLimitInfos = getPostRateLimitInfos(
+    user, 
+    postsInTimeframe, 
+    modRateLimitHours, 
+    manualPostRateLimit, 
+    features
+  );
 
   return getStrictestRateLimitInfo(rateLimitInfos)
 }
 
+const getLongtermScoreInfo = async (userId: string): Promise<LongtermScoreResult> => {
```

Comment:
for consistency's sake, make this a `function`

Reply:
(I think the name should continue to be something like `longtermDownvoteInfo` rather than `longtermScore`, which remove one of the two important bits of information from the name)